doc/api/repl.md

```diff
@@ -433,6 +433,22 @@ buffered but not yet executed. This method is primarily intended to be
 called from within the action function for commands registered using the
 `replServer.defineCommand()` method.
 
+### replServer.setupHistory(historyPath, callback)
```

Comment:
It seems this section should be placed after `### replServer.parseREPLKeyword(keyword[, rest])` section, alphabetically.